tests/unit/test_ps_model.py

```diff
@@ -184,6 +184,22 @@ def test_mach_number_limits():
         mach_num_lim, [0.625, 0.683, 0.750, 0.82, 0.82, 0.82, 0.82], decimal=2
     )
 
+    aircraft_mass = 60000.0
+    mmin = []
+    mmax = []
+    air_temperature = units.m_to_T_isa(units.ft_to_m(altitude_ft))
+    for alt, pres, temp in zip(altitude_ft, air_pressure, air_temperature):
+        mmin.append(ps_lims.minimum_mach_num(pres, aircraft_mass, atyp_param))
+        mmax.append(ps_lims.maximum_mach_num(alt, pres, aircraft_mass, temp, 0.0, atyp_param))
```

Comment:
Does this include cases where the maximum mach number is set by available thrust and cases where it's set by operational limits?